kubernetes_deploy/staging/deployment.yml

```diff
@@ -33,20 +33,25 @@ spec:
         - name: DB_USERNAME
           valueFrom:
             secretKeyRef:
-              name: db
-              key: username
+              name: rds
+              key: database_username
+        - name: DB_NAME
+          valueFrom:
+            secretKeyRef:
+              name: rds
```

Comment:
It seems we could have overwritten the values in the `db` secret. Do we need to introduce a new one?

Reply:
We could have, but I only created the `db` one for this data, so I decided to go with the default naming and just delete the `db` secret.

Reply:
The only reason I have against `rds` is that RDS is "Amazon Relational Database Service" which is very specific to the technology we use, but the configuration stored is generic database configuration (user, password, host, port, database). In this circumstance, I would prefer to call it `db` or `database`.

Reply:
OK that's compelling enough for me. I'll make the change in both places (here and cloud platforms)